test/test_eval.py

```diff
@@ -785,6 +785,7 @@ def test_extractive_qa_eval_wrong_examples(reader, retriever_with_docs):
 
 @pytest.mark.parametrize("retriever_with_docs", ["tfidf"], indirect=True)
 @pytest.mark.parametrize("document_store_with_docs", ["memory"], indirect=True)
+@pytest.mark.parametrize("reader", ["farm"], indirect=True)
```

Comment:
This excludes the `TransformersReader` from this test, which is a good change in my opinion. I was just surprised to find it here. Now that you are at it, please check also the other tests in `test_eval.py`. For example, I see that the same change could be made to `test_extractive_qa_eval_translation`

Reply:
done.